config/crd/bases/exp.infrastructure.cluster.x-k8s.io_azuremachinepools.yaml

```diff
@@ -97,6 +97,30 @@ spec:
               location:
                 description: Location is the Azure region location e.g. westus2
                 type: string
+              maxSurge:
```

Comment:
The `maxSurge` field in the k8s `Deployment` object accepts an absolute number but also a percentage. Should we support percentage as well? I believe it improves the UX as the user doesn't have to adapt the `maxSurge` value when scaling up/down their VMSS. If we decide to support percentages, that would mean changing the type of the field.

Reply:
Oh actually [the proposal talks about both absolute numbers and percentages](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1191/files#diff-e67ed7281d9712d6611f6141a166992ccd15cc63bb553705678d7a6df8831b1bR203-R217).